CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing Guidelines
+
+* You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/).  Each active repo has its own stream to direct questions to (for example #powerup or #portal).  Mentorship System stream topic - Flutter is [#mentorship-flutter](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system/topic/Flutter).
```

Comment:
```suggestion
* You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/).  Each active repo has its own stream to direct questions to (for example #powerup or #portal).  Mentorship System stream is [#mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system).
```

Even though there is a flutter topic, to discuss the project in itself as a product as a whole and its counterparts, you just need to know about the stream. The topics can change over time and all.